ipmininet/router/config/radvd.py

```diff
@@ -0,0 +1,72 @@
+from .base import Daemon
+from .utils import ConfigDict
+from ipmininet.utils import realIntfList
+
+RA_DEFAULT_VALID = 86400
+RA_DEFAULT_PREF = 14400
+DEFAULT_ADV_RDNSS_LIFETIME = 25
+
+
+class RADVD(Daemon):
+    """The base class for all Quagga-derived daemons"""
+
+    # Additional parameters to pass when starting the daemon
+    STARTUP_LINE_EXTRA = ''
+    NAME = 'radvd'
+
+    def __init__(self, *args, **kwargs):
+        super(RADVD, self).__init__(*args, **kwargs)
+
+    def build(self):
+        cfg = super(RADVD, self).build()
+        # Update with preset defaults
+        cfg.update(self.options)
+        cfg.debug = self.options.debug
+        # Track interfaces
+        cfg.interfaces = (self._build_interface(itf)
+                          for itf in realIntfList(self._node))
+        return cfg
+
+    @staticmethod
+    def _build_interface(itf):
+        """Build the ConfigDict object representing the interface"""
+        ra_list = []
+        rdnss_list = []
+        for prefix in itf.ra_prefixes:
+            if prefix.get('prefix') is not None:
+                ra_list.append(
+                    ConfigDict(prefix=prefix['prefix'],
+                               valid_lifetime=prefix.get('valid_lifetime',
```

Comment:
There should be some mention of these parameters somewhere in a docstring, i.e. in IPNet.addLink() (for these and all the others you created earlier)